src/main/java/bio/terra/service/filedata/flight/ingest/IngestDriverStep.java

```diff
@@ -349,6 +367,17 @@ private void launchLoads(
       inputParameters.put(JobMapKeys.CLOUD_PLATFORM.getKeyName(), platform.name());
       inputParameters.put(JobMapKeys.PARENT_FLIGHT_ID.getKeyName(), context.getFlightId());
 
+      // Permissions to view child jobs are inherited from the parent:
+      propagateContextToFlightMap(
+          context,
+          inputParameters,
+          JobMapKeys.IAM_RESOURCE_TYPE.getKeyName(),
```

Comment:
You could avoid the duplicate calls to `getKeyName()` here by having `propagateContextToFlightMap()` accept `JobMapKeys` instead of `String`, and calling `getKeyName()` inside `propagateContextToFlightMap()`.

Reply:
I considered this, but input parameter keys are not always derived from `JobMapKeys`, so I preferred to leave this a bit more generalizable.